homeassistant/components/onvif/device.py

```diff
@@ -37,13 +37,28 @@
     LOGGER,
     PAN_FACTOR,
     RELATIVE_MOVE,
+    EMULATE_RELATIVE_MOVE,
     STOP_MOVE,
     TILT_FACTOR,
     ZOOM_FACTOR,
 )
 from .event import EventManager
 from .models import PTZ, Capabilities, DeviceInfo, Profile, Resolution, Video
 
+def calculate_sleep_time(distance: float, full_rotation_time: float) -> float:
+    """
+    Calculate the sleep time based on the given distance and full rotation time.
+
+    Args:
+        distance (float): The distance to be covered.
+        full_rotation_time (float): The time required for a full rotation in seconds.
+
+    Returns:
+        float: The calculated sleep time in seconds.
+    """
```

Comment:
we generally don't have this kind of docstrings

Reply:
We do allow it though, It's especially encouraged it when it's not obvious from the function name.
However, it's enough to have type information in the function protype, there's no need to duplicate it in the docstring.